src/vsr/grid_scrubber.zig

```diff
@@ -117,8 +117,8 @@ pub fn GridScrubberType(comptime Forest: type) type {
             /// The manifest log tour iterates manifest blocks in reverse order.
             /// (To ensure that manifest compaction doesn't lead to missed blocks.)
             manifest_log: struct { iterator: ManifestBlockIterator = .init },
-            free_set: struct { index: usize = 0 },
-            client_sessions: struct { index: usize = 0 },
+            free_set: struct { index: u32 = 0 },
+            client_sessions: struct { index: u32 = 0 },
```

Comment:
CheckpointTrailer.block_count is u32

Reply:
I originaly hesitated whether i should touch these, but then I noticed that `data_block_index` is u32, so this also turns out a conssitency win if nothing else!